t/jv-allof.t

```diff
@@ -42,7 +42,18 @@ validate_ok 'he110th3re', $schema,
 validate_ok 'hello', {type => ['integer', 'boolean']},
   E('/', 'Expected integer/boolean - got string.');
 
-validate_ok 'hello', {allOf => [true, {type => ['integer', 'boolean']}]},
-  E('/', '/allOf/1 Expected integer/boolean - got string.');
+validate_ok 'hello', {allOf => [{type => ['integer', 'boolean']}]},
+  E('/', '/allOf/0 Expected integer/boolean - got string.');
+
+validate_ok 'hello',
+  {
+  allOf => [
+    {allOf => [{type => 'boolean'}, {type => 'string', maxLength => 2}]},
+    {type  => 'integer'},
+  ],
```

Comment:
Something is up with perltidy here 😞

Reply:
I thought so, but isn't this how you like it? ;/   is the .perltidyrc incorrect?

Reply:
I am definitely using the local .perltidyrc with the latest Perl::Tidy (version 20200110).

Reply:
Sorry, nothing against your code.